windows-agent/cmd/ubuntu-pro-agent/agent/agent_test.go

```diff
@@ -461,3 +462,5 @@ func captureStdout(t *testing.T) func() string {
 		return out.String()
 	}
 }
+
+func TestWithWslSystemMock(t *testing.T) { daemon.WslSystemMock(t) }
```

Comment:
With the changes to the mock, you should not need this anymore, correct?

Reply:
Yeah, instead of doing this, I import `daemontests`, otherwise having that test function is still needed because we need to  test the agent with the command line mock.